docs/quickstart.rst

```diff
@@ -5,28 +5,117 @@ Quickstart
+
+1. Create and configure the Docker Machine {if you don't want to reuse your Docker Host from the Quickstart above} (OS X only).
+
+  .. sourcecode:: bash
+
+    docker-machine create --driver virtualbox --virtualbox-memory 6000 confluent
+
+  Next, configure your terminal window to attach it to your new Docker Machine:
+
+  .. sourcecode:: bash
+
+    eval $(docker-machine env confluent)
+
+2. Clone the CP Docker Images Github Repository.
+
+  .. sourcecode:: bash
+
+    git clone https://github.com/confluentinc/cp-docker-images
+
+  An example Docker Compose file is included that will start up ZooKeeper and Kafka. Navigate to ``cp-docker-images/examples/kafka-single-node``, where it is located.  Alternatively, you can download the file directly from https://github.com/confluentinc/cp-docker-images/raw/master/examples/kafka-single-node/docker-compose.yml 
+
+  .. sourcecode:: bash
+    cd cp-docker-images/examples/kafka-single-node
+
+
+3. Start ZooKeeper and Kafka using Docker Compose ``create`` and ``start`` commands.  You'll run these commands from the directory containing the docker-compose.yml file.
+
+   .. sourcecode:: bash
+
+       docker-compose create
+       docker-compose start
+
+   Before you continue, make sure the services are up and running:
+
+   .. sourcecode:: bash
+
+       docker-compose ps
+
+   You should see the following:
+
+   .. sourcecode:: bash
+
+                  Name                        Command            State   Ports
+       -----------------------------------------------------------------------
+       kafkasinglenode_kafka_1       /etc/confluent/docker/run   Up
+       kafkasinglenode_zookeeper_1   /etc/confluent/docker/run   Up
+
+   Now check the ZooKeeper logs to verify that ZooKeeper is healthy.
+
+   .. sourcecode:: bash
+
+       docker-compose logs zookeeper | grep -i binding
+
+   You should see the following in your terminal window:
+
+   .. sourcecode:: bash
+
+       zookeeper_1  | [2016-07-25 03:26:04,018] INFO binding to port 0.0.0.0/0.0.0.0:32181 (org.apache.zookeeper.server.NIOServerCnxnFactory)
+
+   Next, check the Kafka logs to verify that broker is healthy.
+
+   .. sourcecode:: bash
+
+       docker-compose logs kafka | grep -i started
+
+   You should see message a message that looks like the following:
+
+   .. sourcecode:: bash
+
+       kafka_1      | [2016-07-25 03:26:06,007] INFO [Kafka Server 1], started (kafka.server.KafkaServer)
+
+4. Test the broker by :ref:`following these instructions from Docker Client <test_drive>`.
+
+The confluentinc/cp-docker-images GitHub repository has several other interesting examples of docker-compose.yml files that you can use.
+
```

Comment:
Consider telling people about `docker-compose down`?

Reply:
good point - i see we have similar info for Docker client.